doc/src/build/zk_login.md

```diff
@@ -477,18 +477,26 @@ zkLogin can support providers that work with OpenID Connect built on top of the
 
 Traditional private key wallets demand users to consistently recall mnemonics and passphrases, necessitating secure storage to prevent fund loss from private key compromise.
 
-On the other hand, a zkLogin wallet only requires an ephemeral private key storage with session expiry and the OAuth login flow with expiry. Forgetting ephemeral keys alone does not result in loss of funds, because a user can always sign in again to generate a new ephemeral key and a new ZK proof.
+On the other hand, a zkLogin wallet only requires an ephemeral private key storage with session expiry and the OAuth login flow with expiry. Forgetting an ephemeral key does not result in loss of funds, because a user can always sign in again to generate a new ephemeral key and a new ZK proof.
 
 ## How is zkLogin different from MPC or Multisig wallets?
 
 Multi-Party Computation (MPC) and Multisig wallets rely on multiple keys or distributing multiple key shares and then defining a threshold value for accepting a signature.
 
 zkLogin does not split any individual private keys, but ephemeral private keys are registered using a fresh nonce when the user authenticates with the OAuth provider. The primary advantage of zkLogin is that the user does not need to manage any persistent private key anywhere, not even with any private keys management techniques like MPC or Multisig.
 
-Interestingly though, a zkLogin address automatically simulates a 2-of-2 setting, where the first part is user's OAuth account and the second is user's salt.
+You can think of zkLogin as a 2FA scheme for an address, where the first part is user's OAuth account and the second is the user's salt.
 
 Furthermore, because Sui native supports Multisig wallets, one can always include one or more zkLogin signers inside a Multisig wallet for additional security, such as using the zkLogin part as 2FA in k-of-N settings.
 
+## If my OAuth account is compromised, what happens to my zkLogin address?
+
+Because zkLogin is a 2FA system, an attacker that has compromised your OAuth account cannot access your zkLogin address unless they have separately compromised your password.
```

Comment:
user's salt, not password (actually it's recommended that salts are random 16byte values, it's not a string password)